x-pack/test/reporting_api_integration/reporting_and_security/spaces.ts

```diff
@@ -50,7 +51,35 @@ export default function ({ getService }: FtrProviderContext) {
       // Retry the download URL until a "completed" response status is returned
       const completed$ = getCompleted$(downloadPath);
       const reportCompleted = await completed$.toPromise();
-      expect(reportCompleted).to.match(/^"order_date",/);
+      expect(reportCompleted).to.match(
+        /^"order_date";category;"customer_first_name";"customer_full_name";"total_quantity";"total_unique_products";"taxless_total_price";"taxful_total_price";currency\n"Jul 12, 2019 @ 00:00:00.000";/
```

Comment:
Added more regex to match against so we can verify that timezone is formatted correctly

Reply:
and the separator, which also comes from Advanced Settings (space dependent)